tests/raven/raven_annots/MSD-0003_20180427_5minstart00-15.Table.1.selections.txt

```diff
@@ -0,0 +1,142 @@
+Selection	View	Channel	Begin Time (s)	End Time (s)	Low Freq (Hz)	High Freq (Hz)	Species
```

Comment:
should this file still exist? I assume it is created by a test

Reply:
Yep, this file was added to test `raven.py`'s splitting behavior on a longer/more complicated splitting file